src/main/java/pixelitor/colors/Colors.java

```diff
@@ -132,34 +132,6 @@ private static float hueAverage(float hue1, float hue2) {
         }
     }
 
-    /**
```

Comment:
I don't see any reason to move this method away from here. It's a generic color-related utility method, and this class is for generic color-related utility methods.